packages/core/src/render3/query.ts

```diff
@@ -6,11 +6,12 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
+import * as viewEngine from '../core';
```

Comment:
Add same comment as other file about why this is here?

Reply:
Done (these are going in a follow-up)